src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -49,12 +50,34 @@ object VertexSchema {
   }
 }
 
+object EdgeSchema {
+  val sep = "_"
+  sealed trait EdgeRelationType {
+    def prefix: String
+    def toLabel(label:String) = prefix + sep + label
+  }
+  case object Own extends EdgeRelationType {
+    override val prefix = "OWN"
+  }
+  case object Ref extends EdgeRelationType {
+    override val prefix = "REF"
+  }
+  val allEdgeRelations = Seq(Own, Ref)
+
+  def getEdgeRelation(label: String): EdgeRelationType = {
+    toEdgeRelation(label.split(sep).head)
+  }
+
+  def toEdgeRelation(prefix: String): EdgeRelationType = {
+    allEdgeRelations.filter( er => prefix == er.prefix ).head
+  }
```

Comment:
Instead of all this direct string manipulation, why not have an ```EdgeDescriptor``` case class that contains an ```EdgeRelationType``` and a ```String``` for the suffix, and has a ```toEdgeLabel``` method that joins them?